packages/dds/tree/src/test/shared-tree/sharedTree.spec.ts

```diff
@@ -994,8 +1004,8 @@ describe("SharedTree", () => {
 			assert.deepEqual([...root2], ["A", "B", "C", "y", "D"]);
 
 			provider.processMessages();
-			assert.deepEqual([...tree1.flexTree], expectedAfterRedo);
-			assert.deepEqual([...tree2.flexTree], expectedAfterRedo);
+			assert.deepEqual([...view1.root], expectedAfterRedo);
```

Comment:
for a lot of these comparisons where we spread the arrays, do we have to spread anymore? shouldn't simple tree arrays compare deepequals?

Reply:
That only works if the array schema is in "POJO mode" (i.e. it was defined via `schemaFactory.array(undefined, ...)`). The schema used in this file all have names specified explicitly (`schemaFactory.array("array", ...)`) because the names need to be consistent across different schema (not auto-inferred) for encoding tests and for schema upgrading tests.

Reply:
I've updated the names of the schema to be capitalized, since by convention we use that for the "non-pojo" schema.